src/ripple/shamap/SHAMap.h

```diff
@@ -82,7 +82,7 @@ class SHAMap
     Family&                         f_;
     beast::Journal                  journal_;
     std::uint32_t                   seq_;
-    std::uint32_t                   ledgerSeq_; // sequence number of ledger this is part of
+    std::uint32_t mutable           ledgerSeq_; // sequence number of ledger this is part of
```

Comment:
If we make `setLedgerSeq` non-const and get rid of the `mutable` the code still compiles. I'd get rid of the `mutable`, even if we can justify it. But it's a nit. I'm OK if you still want to keep this.